map-prototype/nodes.json

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "FeatureCollection",
+  "features": [
```

Comment:
Yeah, for a start! tho I think we might eventually want to take advantage of nested key-values for grouping types of info.

Regardless, we can quickly iterate